### ISSUE TITLE
Networks/Network Routes domain routing by TLDs

### ISSUE THREAD

**Issue by fruworg** [netbirdio/netbird#2921]
Hi. It would be great to be able to configure routing by TLDs.
![image](https://github.com/user-attachments/assets/7da287fc-66a1-4a41-b790-2421fb14e192)

**Comment by kinafu** [comment]
There might have been created duplicates of this issue in https://github.com/netbirdio/netbird/issues/2593 and https://github.com/netbirdio/netbird/issues/3782

**Comment by nazarewk** [comment]
@fruworg what is your use case for passing just a TLD to either `Network` `Resource` or `Network Route`?

**Comment by fruworg** [comment]
> [@fruworg](https://github.com/fruworg) what is your use case for passing just a TLD to either `Network` `Resource` or `Network Route`?

Hi!
Bypassing geo-blocks. Some resources are only available through specific exit nodes (based on their location).

**Comment by nazarewk** [comment]
Just to clarify: you mean DNS-based geoblocking?

Wouldn't an Exit Node cover your use case, or do you need more granular routing?

**Comment by fruworg** [comment]
No, some resources are only accessible from specific countries. For example, many Russian websites (.ru TLD) block requests from outside Russia. E.g. rule in Cloudflare:

![Image](https://github.com/user-attachments/assets/363a4af3-0eeb-4621-b306-b15267102afb)
